pymeasure/experiment/results.py

```diff
@@ -69,8 +69,9 @@ def replace_placeholders(string, procedure, date_format="%Y-%m-%d", time_format=
     """
     now = datetime.now()
 
-    parameters = procedure.parameter_objects()
+    parameters = procedure.placeholder_objects()
     placeholders = {param.name: param.value for param in parameters.values()}
+    placeholders.update({varname: param.value for varname, param in parameters.items()})
```

Comment:
does not placeholder_objects already contain the parameters, such that line 73 is superfluous?

Reply:
You are absolutely right; I had removed that but missed it in a commit.

As some context, I was hoping to use some of those nice 'in-string' calculations, like you can do with f-strings (`f"{delay * 10:.2f}"` for example, such that you can format you units nicely for example); for this i would need to add the variable-names of a parameter as a valid placeholder (since the names containing spaces won't work). However, I quickly came to realize that this wont work at all when using the `format` method, rather than f-strings.... unfortunately.